src/components/projects/lean-workflow/EditorElement.tsx

```diff
@@ -124,6 +131,60 @@ const EditorElement: React.FC<Props> = ({ editor }) => {
     },
   })
 
+  const handleAcceptChange = useCallback(
+    (c: TrackedChange) => {
+      const ids = [c.id]
+      if (c.type === 'node-change') {
+        c.children.forEach((child) => {
+          ids.push(child.id)
+        })
+      }
+      execCmd(trackCommands.setChangeStatuses(CHANGE_STATUS.accepted, ids))
+    },
+    [execCmd]
+  )
+  const handleRejectChange = useCallback(
+    (c: TrackedChange) => {
+      const ids = [c.id]
+      if (c.type === 'node-change') {
+        c.children.forEach((child) => {
+          ids.push(child.id)
+        })
+      }
+      execCmd(trackCommands.setChangeStatuses(CHANGE_STATUS.rejected, ids))
+    },
+    [execCmd]
+  )
+  const handleEditorClick = useCallback(
+    (e: React.MouseEvent) => {
+      const button = e.target && (e.target as HTMLElement).closest('button')
+      if (!button) {
+        return
+      }
+      if (!trackState) {
+        return
+      }
+      const { changeSet } = trackState
+
+      const action = button.getAttribute('data-action')
+      const changeId = button.getAttribute('data-changeid')
+
+      if (!action || !changeId) {
+        return
+      } else {
```

Comment:
This else is unneeded, the previous return already stops execution. Although it might work out better to negate the expression and put the else block inside the if